pkg/rpcclient/client.go

```diff
@@ -70,6 +70,13 @@ type Options struct {
 	RequestTimeout time.Duration
 	// Limit total number of connections per host. No limit by default.
 	MaxConnsPerHost int
+	// CloseNotificationChannelIfFull is a WSClient option that allows WSClient to
```

Comment:
WSClient. In a structure that's used for Client. Hmm.

Reply:
Ewe don’t have a separate config for WSClient. Although it’s probably the time to implement it.

Reply:
Fixed.